File: model/session.go
package model

import "time"

type GetSession struct {
	SessionID      int       `json:"sessionID"`
	Title          string    `json:"title"`
	Description    string    `json:"description"`
	SessionOwner   string    `json:"sessinOwner"`
	TimeSlot       time.Time `json:"timeSlots"`
	MaxSeats       int       `json:"maxSeats"`
	AvailableSeats int       `json:"availableSeats"`
	CreatedAt      time.Time `json:"createdAt"`
}

File: src/main/java/conference/entity/Session.java
package conference.entity;

import jakarta.persistence.*;
import java.time.LocalDateTime;

@Entity
@Table(name = "sessions")
public class Session {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "session_id")
    private Long id;

    @Column(name = "title", nullable = false)
    private String title;

    @Column(name = "description")
    private String description;

    @Column(name = "start_time", nullable = false)
    private LocalDateTime startTime;

    @Column(name = "end_time", nullable = false)
    private LocalDateTime endTime;

    @Column(name = "max_seats", nullable = false)
    private Integer maxSeats;

    @Column(name = "available_seats", nullable = false)
    private Integer availableSeats;

    @ManyToOne
    @JoinColumn(name = "created_by", nullable = false)
    private User createdBy;

    public LocalDateTime getStartTime() {
        return startTime;
    }

    public LocalDateTime getEndTime() {
        return endTime;
    }
}
